datafusion/core/src/lib.rs

```diff
@@ -106,7 +106,7 @@
 //! Specifically, when DataFusion receives an SQL query, there are different steps
 //! that it passes through until a result is obtained. Broadly, they are:
 //!
-//! 1. The string is parsed to an Abstract syntax tree (AST) using [sqlparser](https://docs.rs/sqlparser/0.6.1/sqlparser/).
+//! 1. The string is parsed to an Abstract syntax tree (AST) using [sqlparser](https://docs.rs/sqlparser/0.18.0/sqlparser/).
```

Comment:
is there a way we can just link to the latest version?

Reply:
done